asv_bench/benchmarks/libs.py

```diff
@@ -0,0 +1,41 @@
+"""
+Benchmarks for code in pandas/_libs, excluding pandas/_libs/tslibs,
+which has its own directory
+"""
+import numpy as np
+
+from pandas._libs.lib import (
+    is_list_like,
+    is_scalar,
+)
+from pandas import (
+    NA,
+    NaT,
+)
```

Comment:
@MarcoGorelli when the pre-commit hook runs locally, it resets these to single-line imports, which then fails the CI check.  thoughts?

Reply:
@jbrockmendel have you already merged the latest upstream changes? If not, it might be because your `setup.cfg` file didn't have `force_grid_wrap=True` yet?

Reply:
that did it, thanks